app/controllers/api/v1/widget/contacts_controller.rb

```diff
@@ -36,7 +36,8 @@ def destroy_custom_attributes
   def identify_contact(contact)
     contact_identify_action = ContactIdentifyAction.new(
       contact: contact,
-      params: permitted_params.to_h.deep_symbolize_keys
+      params: permitted_params.to_h.deep_symbolize_keys,
+      discard_invalid_attrs: true
```

Comment:
Will it be always true? when are we setting it false in which case?

Reply:
the default value is false in the ContactIdentify action. 
Only in the widget controllers we want this behaviour